### DISCUSSION TITLE
Unable to access `event_loop` or `Window` to get `MonitorHandler`

### DISCUSSION THREAD

**Discussion by yongDataScince** [hecrj/iced#1577] [Q&A]
When creating an application, I need to set the window width to a quarter of the width of the monitor. And also move to the corner of the screen regardless of the screen size.

**Comment by itmecho** [comment]
This would be great as I want to be able to set the width of my window to the primary monitor's width but keep the height at a fixed value.

**Comment by unsecretised** [comment]
Take a look at subscriptions. You can get all resize events and then also use window::move_to that will allow you to move the iced window
